app/models/history.rb

```diff
@@ -24,4 +24,50 @@ class History < ApplicationRecord
                                              'Report Note'] }
 
   belongs_to :patient
+
+  # All histories within the given time frame
+  scope :in_time_frame, lambda { |time_frame|
+    where('histories.created_at >= ?', Time.now - 24.hour) if time_frame == 'Last 24 Hours'
+    where('histories.created_at >= ? AND histories.created_at < ?', Date.today - 14.day, Date.today) if time_frame == 'Last 14 Days'
+  }
+
+  # All histories that are monitoring changes in which a patient is no longer monitored
+  scope :not_monitoring, lambda {
+    where('comment LIKE \'%Not Monitoring%\'')
```

Comment:
Patient model now actually has these things right on them https://github.com/SaraAlert/SaraAlert/blob/master/app/models/patient.rb#L37

Without completely understanding the context here, does it make sense to use that field directly?

Reply:
So the way I'm currently checking for these is by checking the history schema instead of the patient schema because the patient schema only contains information about the current public health action (I imagine some monitorees might have several public health actions over time?).

Reply:
Makes sense